cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/etl/transform/ValidatorTransform.java

```diff
@@ -184,9 +190,19 @@ public void transform(StructuredRecord input, Emitter<StructuredRecord> emitter)
     return new InvalidEntry<>(errorCodeInt, (String) result.get("errorMsg"), input);
   }
 
-  private void init(List<Validator> validators) throws ScriptException {
+  private void init(List<Validator> validators, LookupProvider lookup) throws ScriptException {
     ScriptEngineManager manager = new ScriptEngineManager();
     engine = manager.getEngineByName("JavaScript");
+    try {
```

Comment:
There's quite a lot of repetition among the script transforms. Opened https://issues.cask.co/browse/CDAP-4179 to fix this.